bears/python/PythonPackageInitBear.py

```diff
@@ -0,0 +1,46 @@
+import os
+import re
+
+from coalib.results.Result import Result
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.settings.ConfigurationGathering import (find_user_config)
+
+
+class PythonPackageInitBear(GlobalBear):
+    """
+    Looks for missing __init__.py files in directories containing python files.
+    """
+
+    LANGUAGES = {'Python', 'Python 3', 'Python 2'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+
+    def run(self,
+            ignore_dirs: list = [],
+            create_empty_init_file: bool = True):
+        """
+        :param root_directory:
+            The root directory of the project whose tree has to be checked
+        :ignore_dirs:
+            List of directories to be ignored
+        """
+        files = sorted(self.file_dict)
+        dirs = set()
+        config_path = self.get_config_dir() or find_user_config(
```

Comment:
Why are you even doing this? coala's file_dict should contain everything you need to know about the paths, including ignore paths.